### PR TITLE
Printer friendly view

### PR DESCRIPTION
**Pull request by adulfan**
Print friendly page for CCDB
Sample exports

[massive-filters.pdf](https://github.com/cfpb/ccdb5-ui/files/4299365/massive-filters.pdf)
[complaints-percapita.pdf](https://github.com/cfpb/ccdb5-ui/files/4299366/complaints-percapita.pdf)
[default-complaints.pdf](https://github.com/cfpb/ccdb5-ui/files/4299367/default-complaints.pdf)

To do:

- [x] need to add similar method & css rules for row charts to force the pretty print styles
- [x] dates should be better looking
- [x] update tests

### CODE REVIEW

File: src/PrintInfo.jsx [hunk 1]
```diff
@@ -17,6 +17,7 @@ export class PrintInfo extends React.Component {
   }
 }
 
+/* eslint-disable camelcase */
 export const getComplaintCountText = aggs => {
   const { doc_count, total } = aggs
```
Comment (JeffreyMFarley) [inline src/PrintInfo.jsx]: You can also get around this by "recasting" the name

```javascript
const { doc_count: docCount, total } = aggs
```

Comment (flacoman91) [reply src/PrintInfo.jsx]: thank you. I needed that.

**Comment by JeffreyMFarley** [comment]
Can you provide a sample of the List screen too?  The map ones look 🏅

**Comment by flacoman91** [comment]
Here's a list view one. 

[list-view.pdf](https://github.com/cfpb/ccdb5-ui/files/4299645/list-view.pdf)

**Comment by flacoman91** [comment]
sorry, i need to remove other listeners on the row charts too.

**Comment by flacoman91** [comment]
Ok, i redid the tests and the list and print views both look good.

File: src/PrintInfo.jsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  }
+  return (
+    <div>Showing <span>{ total.toLocaleString() }</span> out of
+      <span> { docCount.toLocaleString() } </span> total complaints </div>
+  )
+}
+
+const getDateText = query => {
+  const { date_received_min: dateMin, date_received_max: dateMax } = query;
+  return shortFormat( dateMin ) + ' - ' + shortFormat( dateMax );
```
Comment (JeffreyMFarley) [inline src/PrintInfo.jsx]: :+1:

**Pull request merged**